CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,217 @@
+
+Behaviors that we consider harassment and exclusionary include but are not limited to:
+
+* Violent threats or language directed against another person
+* Posting sexually explicit or violent material
+* Posting (or threatening to post) someone else's personally identifying information ("doxing")
+* Sharing private content, such as emails sent privately or non-publicly, or direct message history, without the sender’s consent
+* Personal insults, especially those using racist or sexist terms
+* Unwelcome sexual attention
+* Public or private harassment
+* The use of sexualized language or imagery
+* Excessive profanity (please avoid curse words; people differ greatly in their sensitivity to swearing)
+* Sexist, racist, or otherwise discriminatory jokes and language
+* Trolling or insulting and derogatory comments
+* Comments which have the effect of excluding people on the basis of membership in a specific group, including the level of experience, gender, gender identity and expression, sexual orientation, disability, neurotype, personal appearance, body size, race, ethnicity, age, religion, or nationality
+* Persistent unwanted interaction with someone (e.g., messaging, conversation, photography, or recording) after being explicitly asked to stop
+* Other unethical or unprofessional conduct
+* Advocating for, or encouraging, any of the above behaviors
+
+This Code of Conduct is not exhaustive or complete. It serves to capture our common understanding of a productive, collaborative environment. We expect our community members to follow the spirit of the Code of Conduct as much as possible.
+
+## 2. Incident Reporting Guidelines
+
+Code of Conduct incidents reduce the value of our community for everyone.
+Quansight takes reports of misconduct very seriously and is committed to
+preserving and maintaining the welcoming nature of our community and our company
+culture. You are encouraged to be an active bystander especially if you notice a
+dangerous situation or someone in distress.
+
+Should you observe a breach of the Code of Conduct—even if the breach seems
+inconsequential and/or unintentional—you may choose to point the person(s)
+directly to this Code of Conduct. This can happen in private or
+not—whichever is most appropriate. The goal is to treat all parties with dignity
+and respect; ideally, the offending person(s) should have the opportunity to
+correct the behavior and make amends. If you would prefer not to address the
+person(s) directly, please discuss the issue in confidence with a member of the
+COC committee. You can alternatively seek confidential advice from one of the
+Quansight Labs Co-Directors.
+
+In the event of an egregious incident, please report the matter formally—as
+described below—in a timely manner to the entire COC committee (`coc@quansight.com`),
+to any individual member of the COC committee, or to one of the Quansight
```

Comment:
Please remove "to any individual member of the COC committee", that's undesirable. One person can easily miss an email, or mishandle a response.